### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by astridx**
Pull Request for Issue #79 .
### Summary of Changes

Corrected two typos and inserted a warning for Firefox version above 47

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ _We are using `composer.json` file for `tests/codeception` folder, so that you w
 
 ### Run tests
 
-To run the tests please execute the following commands. We are using [Robo.li](http://robo.li/) to execute [PhpUnit](https://phpunit.de/) based [Codeception](http://codeception.com/for/joomla) test suits.
+To run the tests please execute the following commands. We are using [Robo.li](http://robo.li/) to execute [PhpUnit](https://phpunit.de/) based [Codeception](http://codeception.com/for/joomla) test suits. (As we are using Selenium Webdriver 53 at the moment, a Firefox version above 47 might be causing issues.)
```
Comment (javigomez) [inline README.md]: The version is not 53 but 2.53.0

File: README.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ Feature: content
     And I save an article
     Then I should see the "Article successfully saved." message
 ```
-Generate snippets of .featuer file using command `tests/codeception/vendor/bin/codecept gherkin:snippets acceptance`
```
Comment (javigomez) [inline README.md:62]: good catch! ^_^

**Comment by astridx** [comment]
I hope the version are correct now.
In composer.json I found Selenium WebDriver 2.53.1
Before the current firefox version 49 there was version 45. 
With version 45.0.2 I have no problems. With version 49.0.1 I had problems.

**Comment by yvesh** [comment]
@astridx 46, 47.0.1 are also working

**Comment by astridx** [comment]
@yvesh Thank you for correction. But do you now, why this page only shows the versions 45.4 and after that 49? An why my ubuntu only offers my downgrading to version 45.
https://de.wikipedia.org/wiki/Mozilla_Firefox

**Comment by yvesh** [comment]
Thanks @astridx - it's fine that way.

**Pull request merged**